content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -120,3 +124,30 @@ $ cat .dvc/cache/19/6a322c107c2572335158503c64bfba.dir
 ```
 
 That's how DVC knows that the other two cached files belong in the directory.
+
+### Run-cache
+
+`dvc repro` and `dvc run` by default populate and reutilize a log of stage runs
+known, which is found in the `runs/` directory inside the cache (or
+[remote storage](/doc/command-reference/remote)).
+
+Runs are identified as combinations of <abbr>dependencies</abbr>, commands, and
+<abbr>outputs</abbr> that correspond to each other. These combinations are
+hashed into special values that make up the file paths inside the run-cache dir.
```

Comment:
> combinations are hashed into special values

What kind of hash is it, BTW? Probably doesn't matter but I'm curious.

Reply:
Def doesn't matter for the docs themselves, but the details are here https://github.com/iterative/dvc/blob/d05be36694c32507020036db6d3e5e7cc2d45fdb/dvc/stage/cache.py#L48

Reply:
OK cool. Just wanted to confirm it's a hash algo. Resolving

Reply:
Thanks!